src/components/ButtonLink.tsx

```diff
@@ -0,0 +1,20 @@
+// import { useNavigate } from "react-router-dom";
+import { Button, ButtonProps } from "./Button";
+
+type Props = Omit<ButtonProps, "onClick"> & {
+	link: string;
+};
+
+export const ButtonLink = ({}: Props) => {
+	// const navigate = useNavigate();
+	return (
+		<div className="flex space-x-3">
+			<Button onClick={(event) => console.log("klik", event)} type="primary">
```

Comment:
Tady ma byt jeden obecný button ltery ma onClick navigate a ...props. Jak jámě to meli predtim :) tohle je obecná komponenta buttonLink